tools/report.sh

```diff
@@ -48,14 +44,20 @@ while getopts "$optspec" optchar; do
                     usage
                     ;;
             esac;;
-        *)
-            cluster="my-cluster"
-            namespace="myproject"
-            ;;
     esac
 done
 shift $((OPTIND-1))
 
+if [ -z $cluster ]; then
+   echo "Cluster was not specified"
```

Comment:
We should tell them how to do it.

```suggestion
   echo "Cluster was not specified. Use --cluster option to specify it."
```